presentation/src/main/java/com/shifthackz/aisdv1/presentation/activity/AiStableDiffusionActivity.kt

```diff
@@ -47,12 +52,27 @@ class AiStableDiffusionActivity : AppCompatActivity() {
         debugLog("Storage permission is ${result}.")
     }
 
+    @SuppressLint("Recycle")
     override fun onCreate(savedInstanceState: Bundle?) {
-        installSplashScreen()
         super.onCreate(savedInstanceState)
         actionBar?.hide()
-        PermissionUtil.checkNotificationPermission(this, notificationPermission::launch)
-        PermissionUtil.checkStoragePermission(this, storagePermission::launch)
+        val splashScreen = installSplashScreen()
```

Comment:
I suppose this variable should be called before the `super.onCreate()` call.
Did you read the splash screen library [documentation](https://developer.android.com/develop/ui/views/launch/splash-screen/migrate#migrate)?

Reply:
Resolved.